package/boot/grub2/Makefile

```diff
@@ -23,6 +23,10 @@ PKG_BUILD_DEPENDS:=grub2/host
 PKG_ASLR_PIE:=0
 PKG_SSP:=0
 
+ifneq ($(BUILD_VARIANT),none)
+  PKG_TARGETS := bin
```

Comment:
While the intentions are good it removes the package from "opkg list" and therefore hides away the version installed on a running system. I'd keep it as is.